contracts/Oracles/FutarchyOracle.sol

```diff
@@ -63,15 +64,17 @@ contract FutarchyOracle is Oracle {
         uint8 outcomeCount,
         int lowerBound,
         int upperBound,
-        MarketFactory marketFactory,
+        StandardMarketWithPriceLoggerFactory marketFactory,
         MarketMaker marketMaker,
         uint24 fee,
-        uint _deadline
+        uint _deadline,
+        uint startDate
+
     )
         public
     {
         // Deadline is in the future
-        require(_deadline > now);
+        require(_deadline > 0);
```

Comment:
Agree the name is missleading. Is there an easy way to set the timestamp in testrpc to a future timestamp? I found only the wait command, which makes more sense if time-periods are used. Lets rename it to tradingPeriod.